rj_gameplay/rj_gameplay/action/receive.py

```diff
@@ -0,0 +1,34 @@
+"""This module contains the interface and action for receive."""
+
+from abc import ABC, abstractmethod
+
+import stp.role as role
+import stp.action as action
+import stp.rc as rc
+import numpy as np
+from rj_msgs.msg import RobotIntent, SettleMotionCommand
+
+class Receive(action.IAction):
+    """
+    Receive action
+    """
+
+    def __init__(self, robot_id: int = None, one_touch_target:np.ndarray = None):
```

Comment:
For a vanilla receive we don't need one touch target.

There should be a separate action for one touches